packages/form-integrations/test/form-integrations.test.js

```diff
@@ -37,23 +37,51 @@ describe('Form Integrations', () => {
     await el.updateComplete;
     const formEl = el._lionFormNode;
     expect(formEl.formattedValue).to.eql({
-      bio: '',
-      'checkers[]': [['foo', 'bar']],
-      comments: '',
+      full_name: { first_name: '', last_name: '' },
       date: '12/12/2000',
       datepicker: '12/12/2020',
-      dinosaurs: 'brontosaurus',
+      bio: '',
+      money: '',
+      iban: '',
       email: '',
+      checkers: ['foo', 'bar'],
+      dinosaurs: '',
+      favoriteFruit: 'Banana',
+      favoriteMovie: 'Rocky',
       favoriteColor: 'hotpink',
-      full_name: {
-        first_name: '',
-        last_name: '',
-      },
-      iban: '',
       lyrics: '1',
-      money: '',
       range: 2.3,
-      'terms[]': [[]],
+      terms: [],
+      comments: '',
+    });
+  });
+
+  describe('Form Integrations', () => {
+    it('does not become dirty when elements are prefilled', async () => {
+      const el = /** @type {UmbrellaForm} */ (await fixture(
+        html`<umbrella-form
+          .serializedValue="${{
+            full_name: { first_name: '', last_name: '' },
+            date: '2000-12-12',
+            datepicker: '2020-12-12',
+            bio: '',
+            money: '',
+            iban: '',
+            email: '',
+            checkers: ['foo', 'bar'],
+            dinosaurs: 'brontosaurus',
```

Comment:
The prefilled value for 'brontosaurus' currently breaks the test. ('' results in a value of `false` for dirty).
Obviously, solution should be looked for in radioGroup